Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -342,7 +342,7 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
                 readOnly: true,
             });
             this._valueEditor.codeMirror.on("update", () => {
-                this._popover.update();
+                this._popover.update(false);
```

Comment:
Can we make this a named `const` to make it easier to know what we are telling to be false?

Reply:
Done!